### ISSUE TITLE
Unable to run IRkernel with `jupyter-run-repl`

### ISSUE THREAD

**Issue by aNaravi** [nnicandro/emacs-jupyter#305]
The debugger gives this whenever `jupyter-run-repl` is executed (with R kernel) or I try to edit a `jupyter-R` code block:

```
Debugger entered--Lisp error: (error "Customise alist is not specified, nor  ess-local-c...")
  signal(error ("Customise alist is not specified, nor  ess-local-c..."))
  error("Customise alist is not specified, nor  ess-local-c...")
  ess-mode()
  jupyter-repl-mode()
  #f(compiled-function (client &optional repl-name associate-buffer display) #<bytecode 0x157aa3e8d771>)(#<jupyter-org-client jupyter-org-client-157aabfb8314> nil nil nil)
  apply(#f(compiled-function (client &optional repl-name associate-buffer display) #<bytecode 0x157aa3e8d771>) (#<jupyter-org-client jupyter-org-client-157aabfb8314> nil nil nil))
  #f(compiled-function (&rest args) #<bytecode 0x157aa6e80edd>)(#<jupyter-org-client jupyter-org-client-157aabfb8314> nil nil nil)
  apply(#f(compiled-function (&rest args) #<bytecode 0x157aa6e80edd>) #<jupyter-org-client jupyter-org-client-157aabfb8314> (nil nil nil))
  jupyter-bootstrap-repl(#<jupyter-org-client jupyter-org-client-157aabfb8314> nil nil nil)
  jupyter-run-repl("ir" nil nil jupyter-org-client)
  #f(compiled-function (session kernel) "Call `jupyter-run-repl', passing KERNEL as argument." #<bytecode 0x157aa3eaab91>)(#s(org-babel-jupyter-session :name "s") "ir")
  apply(#f(compiled-function (session kernel) "Call `jupyter-run-repl', passing KERNEL as argument." #<bytecode 0x157aa3eaab91>) (#s(org-babel-jupyter-session :name "s") "ir"))
  #f(compiled-function (&rest cnm-args) #<bytecode 0x157aabd199d5>)()
  #f(compiled-function (cl--cnm session kernel) "Rename the returned client's REPL buffer to include SESSION's name.\nAlso set `jupyter-include-other-output' to nil for the session so\nthat output produced by other clients do not get handled by the\nclient." #<bytecode 0x157aa3eaabf1>)(#f(compiled-function (&rest cnm-args) #<bytecode 0x157aabd199d5>) #s(org-babel-jupyter-session :name "s") "ir")
  apply(#f(compiled-function (cl--cnm session kernel) "Rename the returned client's REPL buffer to include SESSION's name.\nAlso set `jupyter-include-other-output' to nil for the session so\nthat output produced by other clients do not get handled by the\nclient." #<bytecode 0x157aa3eaabf1>) #f(compiled-function (&rest cnm-args) #<bytecode 0x157aabd199d5>) (#s(org-babel-jupyter-session :name "s") "ir"))
  #f(compiled-function (&rest args) #<bytecode 0x157aa6baf575>)(#s(org-babel-jupyter-session :name "s") "ir")
  apply(#f(compiled-function (&rest args) #<bytecode 0x157aa6baf575>) #s(org-babel-jupyter-session :name "s") "ir")
  org-babel-jupyter-initiate-client(#s(org-babel-jupyter-session :name "s") "ir")
  org-babel-jupyter-initiate-session-by-key("s" ((:results . "replace") (:exports . "code") (:session . "s") (:async . "no") (:kernel . "ir") (:tangle . "no") (:hlines . "no") (:noweb . "no") (:cache . "no")))
  org-babel-jupyter-initiate-session("s" ((:results . "replace") (:exports . "code") (:session . "s") (:async . "no") (:kernel . "ir") (:tangle . "no") (:hlines . "no") (:noweb . "no") (:cache . "no")))
  org-babel-edit-prep:jupyter-R(("jupyter-R" "import os\nos.getcwd()\n# library(astsa)\n\n# w = rnor..." ((:results . "replace") (:exports . "code") (:session . "s") (:async . "no") (:kernel . "ir") (:tangle . "no") (:hlines . "no") (:noweb . "no") (:cache . "no")) "" nil 24 "(ref:%s)"))
  org-edit-src-code()
  org-edit-special(nil)
  funcall-interactively(org-edit-special nil)
  call-interactively(org-edit-special nil nil)
  command-execute(org-edit-special)
```
On a related issue: I was having similar problems while accessing a remote jupyter R kernel (within a docker container) so I installed r-base and IRkernel on my host. Is it possible to not have jupyter or R installed on the host and still access a running remote jupyter kernel?

**Comment by fleimgruber** [comment]
Regarding your traceback: how did you configure emacs-jupyter for jupyter-R? I just tried with emacs-jupyter 6ce8d01e3a550a3268b415bf9d9b635d4dba5940 and this config

```emacs-lisp
(setq org-babel-default-header-args:jupyter-R '((:async . "yes")
                                                (:session . "r")
                                                (:kernel . "ir")))
```

where the `:kernel` part is the default name that `IRKernel` chooses when it is installed with `IRkernel::installspec()`.
This src block

```
#+begin_src jupyter-R :display text/plain
rnorm(10)
#+end_src
```

```
#+RESULTS:
:  [1]  0.98940247 -1.77209916 -0.48638323  1.78818407  0.15950350 -0.16047767
:  [7]  0.52812129 -0.01973934 -0.17240960  2.29112216
```

and `jupyter-run-repl` are working.

Regarding the related issue: you should be able to access a remote jupyter R kernel without having to have R installed on the client - you do need to have `jupyter` installed on the client though.

**Comment by aNaravi** [comment]
Thanks for the reply, this configuration works for me now.

I'm not sure why it wasn't working before. Its been a while since I encountered this error: I had stopped using `jupyter-R` and had removed those lines from the configuration. However, since then I have started using a new package manager, [straight](https://github.com/raxod502/straight.el), and have upgraded all my packages to their latest versions. I was previously using melpa-stable versions of several packages. 

That seems to have done the trick. I'm also able to access remote jupyter kernels.
